templates/tour.html

```diff
@@ -4,300 +4,301 @@
+                    <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-stp" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="p-link--external" aria-label="External link to install and configure spanning tree protocol on MAAS documentation"><abbr title="Spanning tree protocol">STP</abbr></a></li>
+                    <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-proxy" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="p-link--external" aria-label="External link to install and configure network proxy on MAAS documentation">Proxy</a></li>
+                </ul>
         </div>
-    </section>
-    <section class="row strip-dark strip-dark--fluid">
-        <div class="inner-wrapper">
-            <div class="six-col strip-dark__col equal-height">
-                <div class="two-col">
-                    <img src="{{ ASSET_SERVER_URL }}4508bba8-maas-cli.svg" class="image--shadow" alt="MAAS command line interface icon">
-                </div>
-                <div class="four-col last-col">
-                    <h3><a href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" aria-label="External link to manage command line interface page on MAAS documentation">See MAAS <abbr title="Command line interface">CLI</abbr> on how to get started with the <abbr title="Command line interface">CLI</abbr>&nbsp;&rsaquo;</a></h3>
-                </div>
+    </div>
+</section>
+
+<section class="p-strip--accent is-deep">
+    <div class="row">
+        <div class="col-6 u-equal-height">
+            <div class="col-2">
+                <img src="{{ ASSET_SERVER_URL }}4508bba8-maas-cli.svg" alt="MAAS command line interface icon" />
             </div>
-            <div class="six-col last-col strip-dark__col strip-dark__col--light equal-height">
-                <div class="two-col">
-                    <img src="{{ ASSET_SERVER_URL }}a92a0e74-maas-api.svg" class="image--shadow" alt="MAAS application programming interface icon">
-                </div>
-                <div class="four-col last-col">
-                    <h3><a href="https://docs.ubuntu.com/maas/2.2/en/api" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" aria-label="External link to application programming interface page on MAAS documentation">See the <abbr title="Application programming interface">API</abbr> documentation to automate and extend MAAS&nbsp;&rsaquo;</a></h3>
-                </div>
+            <div class="col-4">
```

Comment:
This looks like it can be achieved with the heading icon pattern. 
https://github.com/vanilla-framework/vanilla-brochure-theme/blob/develop/docs/patterns/_heading-icon.md